changelog.d/0-release-notes/background-worker

```diff
@@ -0,0 +1,38 @@
+This release introduces a new component: background-worker. This is currently
+only used to forward notifications to federated backends. Enabling federation in
+the wire-server helm chart automatically installs this component.
+
+When federation is enabled, wire-server will require running RabbitMQ. The helm
+chart in `rabbitmq` can be used to install RabbitMQ. Please refer to the
+documentation at https://docs.wire.com to install RabbitMQ in Kubernetes. These
+new configurations are required:
```

Comment:
<plsignoreimtalkingtomyself>move the yaml snippet to docs.wire.com?  or if it's already there: link to it rather than redundantly copy it?  on the other hand, that would complicate the situation in which we change docs.wire.com in a later release, and somebody upgrades to this release afterwards.</plsignoreimtalkingtomyself>